Python/Product/Uwp/Interpreter/PythonUwpInterpreterFactoryProvider.cs

```diff
@@ -23,39 +25,68 @@ namespace Microsoft.PythonTools.Uwp.Interpreter {
     class PythonUwpInterpreterFactoryProvider : IPythonInterpreterFactoryProvider {
         private HashSet<IPythonInterpreterFactory> _factories = null;
 
-        public const string DefaultVersion = "3.5";
-
         public event EventHandler InterpreterFactoriesChanged;
 
         public PythonUwpInterpreterFactoryProvider() {
-            PythonVersion = DefaultVersion;
-        }
-
-        public string PythonVersion {
-            get; set;
         }
 
         private void DiscoverFactories() {
             if (_factories == null) {
-                var defaultConfig = new InterpreterConfiguration(
-                    null,
-                    null,
-                    null,
-                    null,
-                    null,
-                    ProcessorArchitecture.None,
-                    new Version(PythonVersion),
-                    InterpreterUIMode.CannotBeDefault
-                );
+                var userSdkInstallDir = new DirectoryInfo(Environment.ExpandEnvironmentVariables(@"%LOCALAPPDATA%\Microsoft\VisualStudio\%VISUALSTUDIOVERSION%Exp\Extensions\Microsoft\Python UWP"));
+                var sdkInstallDir = new DirectoryInfo(Environment.ExpandEnvironmentVariables(@"%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0\ExtensionSDKs\Python UWP"));
 
                 _factories = new HashSet<IPythonInterpreterFactory>();
 
-                _factories.Add(new PythonUwpInterpreterFactory(defaultConfig));
-
-                var factoriesChanged = InterpreterFactoriesChanged;
-
-                if (factoriesChanged != null) {
-                    factoriesChanged(this, new EventArgs());
+                if (userSdkInstallDir.Exists) {
+                    foreach (var dirInfo in userSdkInstallDir.EnumerateDirectories()) {

```

Comment:
I also don't see any particular reason not to just iterate over `userSdkInstallDir.EnumerateFiles("CPython.targets", SearchOption.AllDirectories)` here, rather than doing a recursive search within each directory. Combined with ensuring the directory name is a valid version number, that will exclude any top-level .targets files, which would seem to be the only difference.